test/e2e/cmd/workspaces_test.go

```diff
@@ -56,19 +56,19 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	//TODO: Have better improvement of errors.
```

Comment:
Does this PR do improvement that this TODO means?

Reply:
yes, I'll delete the comment.